### PR TITLE
Add initial CRDS notebooks

### PR DESCRIPTION
**Pull request by eslavich**
This PR adds four notebooks that provide an introduction to CRDS, with particular attention to .rmaps and file submission.  They'll need some minor updates after I fix a couple of small issues in the CRDS client, but for the most part they're ready to go.

### CODE REVIEW

File: crds/02_basic_file_submission.ipynb [hunk 1]
```diff
@@ -0,0 +1,509 @@
...
+    "https://jwst-crds-bit.stsci.edu/\n",
+    "\n",
+    "and, in the upper-right corner, click the login button:\n",
+    "\n",
+    "![Login button](images/login.png)\n",
+    "\n",
+    "The login page will ask you to select an instrument to \"lock\".  This lock grants you exclusive access to that instrument while making your submission.  Note that this means that multiple users cannot complete this notebook simultaneously.\n",
+    "\n",
+    "Select \"miri\" and press `Continue`.  You will be asked to identify yourself with your STScI SSO credentials, and perhaps also to authorize CRDS to access details of your SSO identity.\n",
+    "\n",
```
Comment (nden) [inline crds/02_basic_file_submission.ipynb:273]: I am not asked to authenticate with SSO but I see 
```
CRDS is requesting access to:

Access your user profile data - mast:user:info
```
and an `Authorize` button below. I'm not on VPN, as I should be.

Comment (nden) [reply crds/02_basic_file_submission.ipynb:273]: And I see my user name at the top right corner. Is this expected?

Comment (eslavich) [reply crds/02_basic_file_submission.ipynb:273]: It sounds like you were already signed into SSO.  The second half of that sentence (`and perhaps also to authorize CRDS to access details of your SSO identity.`) refers to the screen that you're now facing.  Maybe I should add more screenshots?

**Comment by nden** [comment]
I read the notebooks and did most of the exercises. 👍 to merge. Thanks Ed!

**Review by nden — APPROVED** [review]

**Review by perrygreenfield — APPROVED** [review]
I didn't do the exercises, but I also approve.

**Pull request merged**